### DISCUSSION TITLE
Automatic re-evaluation of :if, :inapt-if

### DISCUSSION THREAD

**Discussion by kickingvegas** [magit/transient#289] [Support] [answered]
Is there a way to re-evaluate `:inapt-if` for a menu item if a command on that menu changes its value and the Transient menu is kept raised? This works automatically for changes in value to `:description`, it would be nice to see that automatic behavior apply to `:inapt-if` and the `:if` family of keys.

**Comment by tarsius** [comment, marked as answer]
Set `:refresh-suffixes` to t in the prefix definition.

**Reply by kickingvegas** [reply]
Thanks for the answer!
